sdk/nodejs/ec2/subnet.ts

```diff
@@ -97,7 +97,7 @@ export class Subnet extends pulumi.CustomResource {
      * The IPv6 network range for the subnet,
      * in CIDR notation. The subnet size must use a /64 prefix length.
      */
-    public readonly ipv6CidrBlock!: pulumi.Output<string>;
+    public readonly ipv6CidrBlock!: pulumi.Output<string | undefined>;
```

Comment:
This could be source breaking if someone uses this within an apply and doesn't account for undefined. Is there a way to avoid this change?

Reply:
There is not - as this is an output we can't do anything about undefined params :/